log_analysis/http/index.html

```diff
@@ -0,0 +1,85 @@
+<html>
+<head>
+<style>
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+  position:absolute;
+  top: 30px;
+}
+
+td, th {
+  border: 1px solid;
+  text-align: left;
+  padding: 8px;
+}
+
+th:nth-child(odd) {
+  background-color: #dddddd;
+}
+
+#error { 
+  color: red;
+  border: black 1px solid;
+}
+</style>
+</head>
+
+<script>
+function selectElement(id, valueToSelect) {
+    let element = document.getElementById(id);
+    element.value = valueToSelect;
+}
+function hideTables() {
+   var ids = ["augur", "gunicorn", "housekeeper", "github_worker", "facade_worker", "linux_badge_worker", "pull_request_worker", "repo_info_worker", "release_worker"];
+   var i;
+   for (i = 0; i < ids.length; i ++) {
+       var elements = document.getElementsByClassName(ids[i]);
+       var j;
+       for (j = 0; j < elements.length; j++) {
+           elements[j].style.visibility="hidden";
+       }
+   }
+}
+function myFunction() {
+    hideTables()
+    var input, filter, table, tr, i;
+    input = document.getElementById("jobs_list");
+    filter = input.value;
+    var elements = document.getElementsByClassName(filter);
+    var slideOpen = true;
+    var initHeight = 0;
+    var previousOffset = 100;
+    for (i = 0; i < elements.length; i++) {
+        elements[i].style.visibility="visible";
+        if(slideOpen) {
+            slideOpen = false;
+            elements[i].style.top = '100px';
+            previousOffset = previousOffset + 60;
+        }
+        else {
+            initHeight = 100 + 60 * i + previousOffset;
+            previousOffset = previousOffset + elements[i].offsetHeight;
+            elements[i].style.top = initHeight + 'px';
+        }
+    }
+}
+</script>
+<h2>Select a worker:</h2>
+<select id="jobs_list" onchange="myFunction()" class='form-control'>
+<option value="none">-- Select --</option>
+<option value="github_worker">github_worker</option>
+<option value="facade_worker">facade_worker</option>
+<option value="linux_badge_worker">linux_badge_worker</option>
+<option value="pull_request_worker">pull_request_worker</option>
+<option value="release_worker">release_worker</option>
+<option value="repo_info_worker">repo_info_worker</option>
+</select>
+</html>
```

Comment:
This code seems functionally very similar to what's in `empty_index.html`. Is there any way to consolidate the logic so there is no code unnecessarily repeated?

Reply:
@ccarterlandis this page is being populated while running logstash and `empty_index.html` is the "backup", i.e. the empty file that contains no errors data. I found it better to do a cleanup this way, rather than delete specific lines from `index.html`, as the latter may result in unexpected issues (f.e. if someone changes the script, or adds new workers and the line numbers are shifted forward, they will need to update the code, so that the cleanup doesn't delete lines accidentally. 
May be it's better to generate `index.html` during execution and add it to `.gitignore`.

Reply:
Ah okay, I see what you were going for. Since these are pretty small pages, and this is an MVP, I think that your solution is fine. We can additional functionality as we need it, and re-evaluate if we want to change how this is implemented then.